widget/ui/src/components/SelectableCategoryList/SelectableCategoryList.helpers.ts

```diff
@@ -19,17 +32,26 @@ const filterByType = (blockchain: BlockchainMeta, type: string): boolean => {
   }
 };
 
-const blockchainslogo = {
-  [BlockchainCategories.EVM]: ['ETH', 'BSC', 'POLYGON'],
-  [BlockchainCategories.COSMOS]: ['COSMOS', 'OSMOSIS', 'JUNO'],
-  [BlockchainCategories.UTXO]: ['BTC', 'LTC', 'BCH'],
-  [BlockchainCategories.OTHER]: ['STARKNET', 'TRON', 'SOLANA'],
-};
-
-export const generateBlockchainsLogo = (
+export const hasAnyCategory = (
   list: BlockchainMeta[],
-  type: Exclude<BlockchainCategories, 'ALL'>
-) =>
-  list
-    .filter((item) => filterByType(item, type))
-    .filter((item) => blockchainslogo[type].includes(item.name));
+  blockchainType: string
+) => list.some((blockchain) => filterByType(blockchain, blockchainType));
+
+export const getCountCategories = (list: BlockchainMeta[]) => {
```

Comment:
In order to hide the section, we use many loops. It might not be worth it. what do you think?